dev-support/docker/Dockerfile_aarch64

```diff
@@ -144,7 +144,7 @@ RUN npm install -g bower@1.8.8
 ###
 # Avoid out of memory errors in builds
 ###
-ENV MAVEN_OPTS -Xms256m -Xmx1536m
+ENV MAVEN_OPTS -Xms256m -Xmx3072m
```

Comment:
For aarch64 trunk doesn't have this change, worth pushing there as well.

Reply:
Thanks for the review. Good idea. I requested your review on #5522 for this.